lib/reek/ast/sexp_extensions/send.rb

```diff
@@ -0,0 +1,52 @@
+module Reek
+  module AST
+    module SexpExtensions
+      # Utility methods for :send nodes.
+      module SendNode
+        VISIBILITY_MODIFIERS = [:private, :public, :protected, :module_function]
+        ATTR_DEFN_METHODS = [:attr_writer, :attr_accessor]
+
+        def receiver; children.first; end
+        def method_name() children[1]; end
+        def args() children[2..-1] end
```

Comment:
That there were three one-line method definitions and every single one of them used a different syntax: first with two semicolons, second with parentheses and a semicolon, and the third one with parentheses and no semicolon. A great use-case for parsing, not so great for consistency. ;)

Thanks for unwrapping them!